index.js

```diff
@@ -236,10 +243,18 @@ function decode(encoded) {
         data.headers = encoded;
     }
 
+    var dataHeaders = data.headers;
+
     try {
         data.headers = JSON.parse(data.headers);
-    } catch(e) {
-        throw new Error('Invalid cache value');
+    } catch (e) {
```

Comment:
Could we add the `e` to the log message here?

Reply:
Yeah, this would be good